src/bokeh/sampledata/us_marriages_divorces.py

```diff
@@ -66,7 +66,7 @@ def _read_data() -> DataFrame:
     '''
 
     data = package_csv('us_marriages_divorces', 'us_marriages_divorces.csv')
-    return data.interpolate(method='linear', axis=0).ffill().bfill()
+    return data.interpolate(method='linear', axis=0, inplace=False).ffill().bfill()
```

Comment:
Explicit `inplace=False` forces the type checker to use the correct return type. So maybe it's not a regression in `pandas-stubs` but mypy.

Reply:
> but mypy

This job uses the exact same 0991 version of mypy as all the previous recently passing jobs, AFAICT

Reply:
Yes, it's not mypy. Last working version of pandas-stubs is v1.5.2.221213 (previous version), so this was broken in yesterday's release.